Mathlib/NumberTheory/AbelSummation.lean

```diff
@@ -282,12 +283,12 @@ theorem tendsto_sum_mul_atTop_nhds_one_sub_integral₀ (hc : c 0 = 0)
   have h_lim' : Tendsto (fun n : ℕ ↦ ∫ t in Set.Ioc (1 : ℝ) n, deriv f t * ∑ k ∈ Icc 0 ⌊t⌋₊, c k)
       atTop (𝓝 (∫ t in Set.Ioi 1, deriv f t * ∑ k ∈ Icc 0 ⌊t⌋₊, c k)) := by
     refine Tendsto.congr' h (intervalIntegral_tendsto_integral_Ioi _ ?_ tendsto_natCast_atTop_atTop)
-    exact integrableOn_Ici_iff_integrableOn_Ioi.mp
+    refine integrableOn_Ici_iff_integrableOn_Ioi.mp
```

Comment:
Why this change?

Reply:
Good question. Probably an oversight. Thanks!